apps/api/src/modules/resume/resume.controller.ts

```diff
@@ -58,4 +67,16 @@ export class ResumeController {
   getResume(@Param('resumeId') resumeId: number, @User() user: JwtDto) {
     return this.resumeService.getResumeById(+resumeId, user)
   }
+
+  @Delete(':resumeId')
+  @HttpCode(204)
+  @UseAuthGuard(UserType.ACTOR)
+  @ApiResponse({ status: 204, description: 'Delete resume complete.' })
```

Comment:
```suggestion
  @ApiNoContentResponse({ description: 'Delete resume complete.' })
```
use @ApiNoContentResponse instead